build_npm.ts

```diff
@@ -7,7 +7,11 @@ await copy("testdata", "npm/umd/testdata", { overwrite: true });
 await copy("README.md", "npm/README.md", { overwrite: true });
 
 await build({
-  entryPoints: ["./main.ts"],
+  entryPoints: ["./lib.ts", {
```

Comment:
Is it correct to have "./lib.ts" as an entrypoint for someone who wants to use the API?

The package.json looks like this now:

```json
{
  "module": "./esm/lib.js",
  "main": "./umd/lib.js",
  "types": "./types/lib.d.ts",
  "bin": {
    "license_checker": "./umd/main.js"
  },
  "name": "@kt3k/license-checker",
  "version": "3.1.4",
  "description": "📄 CLI tool for checking license headers in files",
  "license": "MIT",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/kt3k/deno_license_checker.git"
  },
  "bugs": {
    "url": "https://github.com/kt3k/deno_license_checker/issues"
  },
  "exports": {
    ".": {
      "import": "./esm/lib.js",
      "require": "./umd/lib.js",
      "types": "./types/lib.d.ts"
    }
  },
  "scripts": {
    "test": "node test_runner.js"
  },
  "dependencies": {
    "tslib": "2.3.1",
    "deno.ns": "0.6.4"
  },
  "devDependencies": {
    "@types/node": "16.11.1",
    "chalk": "4.1.2"
  }
}
```